client/app/components/dashboards/ExpandWidgetDialog.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'antd/lib/button';
+import Modal from 'antd/lib/modal';
+import { VisualizationRenderer } from '@/visualizations/VisualizationRenderer';
+import { wrap as wrapDialog, DialogPropType } from '@/components/DialogWrapper';
+import { VisualizationName } from '@/visualizations/VisualizationName';
+
+function ExpandWidgetDialog({ dialog, widget }) {
```

Comment:
Took me some time to understand what this name means.
Wdyt of:
a) `ExpandedWidgetDialog`
b) `FullscreenWidgetDialog`
c) `FullSizeWidgetDialog`
d) `FullViewWidgetDialog`